### PR TITLE
New update skeleton and class builder

### PR DESCRIPTION
**Pull request by FHenry**

### CODE REVIEW

File: dev/skeletons/skeleton_class.class.php [hunk 1]
```diff
@@ -175,6 +178,69 @@ function fetch($id,$ref='')
...
+    		
+    		while ($obj = $this->db->fetch_object($resql))
+    		{
+    			$line=new Skeleton_ClassLine();
+    
+    			$line->id    = $obj->rowid;
+    			$line->prop1 = $obj->field1;
+    			$line->prop2 = $obj->field2;
+    			
+    			$this->line[]=$line;
```
Comment (jfefe) [inline dev/skeletons/skeleton_class.class.php:229]: Not $this->lines here?

File: dev/skeletons/skeleton_class.class.php [hunk 2]
```diff
@@ -175,6 +178,69 @@ function fetch($id,$ref='')
...
+     *  Load object in memory from the database
+     *
+     * @param string $sortorder Sort Order
+	 * @param string $sortfield Sort field
+	 * @param int $limit offset limit
+	 * @param int $offset offset limit
+	 * @param array $filter filter array
+	 * @return int <0 if KO, >0 if OK
+     */
+    function fetchAll($sortorder, $sortfield, $limit, $offset, $filter = array())
```
Comment (jfefe) [inline dev/skeletons/skeleton_class.class.php:191]: Could you add $user parameter to allow permissions test into this method? 
Or it's better to use global $user ?

Comment (FHenry) [inline dev/skeletons/skeleton_class.class.php:191]: Most read permission are done in card or list pages, eg fetch method (no user rights test)

File: dev/skeletons/skeleton_class.class.php (hunk 1 above)
Comment (FHenry) [inline dev/skeletons/skeleton_class.class.php:229]: yes true, I send the pull request

**Pull request merged**